AdyenNetworking/APIClient/RetryAPIClient.swift

```diff
@@ -69,17 +69,24 @@ public final class RetryAPIClient: AnyRetryAPIClient {
     }
     
     /// :nodoc:
-    public func perform<R>(downloadRequest request: R, completionHandler: @escaping CompletionHandler<R.ResponseType>) where R : Request, R.ResponseType == DownloadResponse {
-        perform(downloadRequest: request, shouldRetry: nil, completionHandler: completionHandler)
+    public func perform<R>(
```

Comment:
we probably don't need this function, and the next one as well. one implementation should be enough